feathr_project/feathr/_feature_registry.py

```diff
@@ -578,3 +579,42 @@ def list_registered_features(self, project_name: str = None) -> List[str]:
                 feature_list.append(entity["name"])
 
         return feature_list
+    
+    def sync_features_from_registry(self, project_name: str, workspace_path: str):
```

Comment:
Should we say something like "get_features_from_registry" so that it's clear on the direction?

Reply:
Good point, addressed.